src/client/datascience/kernel-launcher/localKernelFinder.ts

```diff
@@ -90,12 +91,13 @@ export class LocalKernelFinder implements ILocalKernelFinder {
         try {
             // Get list of all of the specs
             const kernels = await this.listKernels(resource, cancelToken);
+            const isPythonNb = isPythonNotebook(option);
 
             // Always include the interpreter in the search if we can
             const interpreter =
                 option && isInterpreter(option)
                     ? option
-                    : resource && this.extensionChecker.isPythonExtensionInstalled
+                    : resource && isPythonNb && this.extensionChecker.isPythonExtensionInstalled
```

Comment:
Use active interpreter only if we know this is a python notebook

Reply:
Not sure this works with the interactive window

Reply:
Aaah, i think i can use the resource to check that, thanks

Reply:
Fixed.